sql_metadb/report_queries/erm/resource_lists/erm_agreement_cancellation_dates.sql

```diff
@@ -0,0 +1,59 @@
+-- Creates a report that contains an overview of agreements and their cancellation deadlines.
+--
+-- For the filters in parameter, only 'sao role' is mandatory.
+-- The rest can be omitted if the where clause is commented out.
+-- The filters can be used, for example, to display agreements whose cancellation deadline is within a certain period of time.
+WITH parameters AS (
+    SELECT
+        'vendor' AS sao_role, -- Is required to restrict the table column "agreement_vendor" to vendor.
+        '2022-01-01'::date AS agreement_cancellation_interval_start, -- start date for cancellation interval, e.g. 2022-01-01
+        '2022-12-31'::date AS agreement_cancellation_interval_end, -- end date for cancellation interval, e.g. 2022-12-31
+        'active' AS agreement_status,
```

Comment:
In other report queries we had given some examples what vaules can be used in the filters for more convenience. Perhaps we could provide some of the reference vaules here too. 
eg. `-- Enter your agreement_status eg. 'active', 'closed', 'draft', 'requested' etc.`